src/main/scala/chisel3/util/Counter.scala

```diff
@@ -5,10 +5,12 @@ package chisel3.util
 import chisel3._
 import chisel3.internal.naming.chiselName  // can't use chisel3_ version because of compile order
 
-/** A counter module
+/** Used to generate an inline hardware counter
   *
   * Typically instantiated with apply methods in [[Counter$ object Counter]]
   *
+  * Does not create a new Chisel Module
```

Comment:
You could merge this with the above sentence: 
`Used to generate an inline (logic directly in the containing Module, no internal Module is created) hardware counter.`

Reply:
Done. Commit added to PR